app/serializers/dist/json_serializer.rb

```diff
@@ -70,5 +79,56 @@ def serialize_nested(connection)
         name: connection.name,
       }
     end
+
+    def build_mapping_blocks
+      puts "Generating mappings ..."
```

Comment:
Do we need this in our serializers? I think it's reasonable for serializers to not have console output 😅